sdk/storage/azure-storage-queue/src/main/java/com/azure/storage/queue/QueueServiceSASSignatureValues.java

```diff
@@ -14,7 +14,7 @@
 /**
```

Comment:
Rename this class as well?

Reply:
Yes, I will have a separate PR for renaming all acronym related classes to minimize merge conflicts.